src/status_im/utils/handlers.cljs

```diff
@@ -29,9 +29,9 @@
     :after
     (fn check-handler
       [context]
-      (let [new-db (get-coeffect context :db)
+      (let [new-db (get-effect context :db)
             v (get-coeffect context :event)]
-        (when-not (s/valid? :status-im.specs/db new-db)
+        (when (and new-db (not (s/valid? :status-im.specs/db new-db)))
```

Comment:
Right this sounds great. Re-frame is doing [this internally](https://github.com/Day8/re-frame/blob/cf78ad43614b3cd250f8a935a9252f25551a048f/src/re_frame/std_interceptors.cljc#L50).